### PR TITLE
Completed css-flexbox

### PR DESCRIPTION
**Pull request by amorabryan**
image
image
image
image

### CODE REVIEW

File: css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        flex-basis: calc(100% / 6);
```
Comment (shawnkost) [inline css-flexbox/flex-wrap.html]: `flex-basis` is a child property, meaning it gets applied to flex children, and not the flex container itself. In this case, it needs to get applied to `.card-wrapper`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Bryan!

**Comment by amorabryan** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**